integrationTests/relayers/ethToMultiversXWithChainSimulator_test.go

```diff
@@ -157,178 +162,467 @@ type keysHolder struct {
 	ethAddress common.Address
 }
 
-type testConfig struct {
-	isMintBurnOnMvX bool
-	isNativeOnMvx   bool
-	isMintBurnOnEth bool
-	isNativeOnEth   bool
+func TestRelayersShouldExecuteTransfers(t *testing.T) {
+	t.Run("ETH->MVX and back, ethNative = true, ethMintBurn = false, mvxNative = false, mvxMintBurn = true", testRelayersShouldExecuteTransfersEthToMVX(
+		argSimulatedSetup{
+			mvxHexIsMintBurn: hexTrue,
+			mvxHexIsNative:   hexFalse,
+			ethIsMintBurn:    false,
+			ethIsNative:      true,
+		},
+	))
+	t.Run("MVX->ETH, ethNative = false, ethMintBurn = true, mvxNative = true, mvxMintBurn = false", testRelayersShouldExecuteTransfersMVXToETH(
+		argSimulatedSetup{
+			mvxHexIsMintBurn: hexFalse,
+			mvxHexIsNative:   hexTrue,
+			ethIsMintBurn:    true,
+			ethIsNative:      false,
+		},
+	))
 }
 
-func TestTransfersBothWaysWithChainSimulator(t *testing.T) {
-	t.Run("Eth: native, MvX: mint & burn", func(t *testing.T) {
-		cfg := testConfig{
-			isMintBurnOnMvX: true,
-			isNativeOnMvx:   false,
-			isMintBurnOnEth: false,
-			isNativeOnEth:   true,
+func testRelayersShouldExecuteTransfersEthToMVX(argsSimulatedSetup argSimulatedSetup) func(t *testing.T) {
```

Comment:
to reduce the indentation of this function (remove the return of an anonymous function), the code can look like:
```GO
func testRelayersShouldExecuteTransfersEthToMVX(t *testing.T, argsSimulatedSetup argSimulatedSetup) {
 // function body
```
and when called in a test, can be expanded as:
```GO
args := argSimulatedSetup{
    mvxHexIsMintBurn: hexTrue,
    mvxHexIsNative:   hexTrue,
    ethIsMintBurn:    true,
    ethIsNative:      false,
}

testRelayersShouldExecuteTransfersMVXToETH(t, args)
```

Reply:
applied